README.md

```diff
@@ -56,48 +56,44 @@ pnpm i @thinknimble/tn-models-fp
 ## Quickstart
 
 ```typescript
-
 /**
  *
  *  // creating a simple user api with login, registration, update
  *
  */
 
-
-
-
-import axios from 'axios' // it is not required to use axios - pick any client
-import {z} from 'zod'
-import {GetInferredFromRaw, createCustomServiceCall } from '@thinknimble/tn-models-fp'
+import axios from "axios" // it is not required to use axios - pick any client
```

Comment:
oh, no axios actually is required. It is a peer dep as well as zod.
We're relying on `params` for query building as well as the http method calls from it.
I thought about making this client agnostic but didn't go for it due to how much more abstract and complicated things would be.
We can take this to a discussion and grind on what approaches we could take for this matter. #90